django_project/feti/static/feti/css/landing-page.css

```diff
@@ -546,7 +546,7 @@
 .result-title img {
     width: 54px;
     height: auto;
-    margin-right: 20px;
+    margin: 10px 5px 5px 5px;
```

Comment:
hi @ann26 
i'm not sure it is the solution.
the margin is already good, but the problem is when icon's height is not big enough, it will fall in top of div. What we need it center it.